protocols/v2/framing-sv2/README.md

```diff
@@ -0,0 +1,60 @@
+# `framing_sv2`
+
+[![crates.io](https://img.shields.io/crates/v/framing_sv2.svg)](https://crates.io/crates/framing_sv2)
+[![docs.rs](https://docs.rs/framing_sv2/badge.svg)](https://docs.rs/framing_sv2)
+[![rustc+](https://img.shields.io/badge/rustc-1.75.0%2B-lightgrey.svg)](https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html)
+[![license](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)](https://github.com/stratum-mining/stratum/blob/main/LICENSE.md)
+[![framingov](https://framingov.io/gh/stratum-mining/stratum/branch/main/graph/badge.svg?flag=framing_sv2-coverage)](https://framingov.io/gh/stratum-mining/stratum)
+
+`framing_sv2` provides utilities for framing messages sent between Sv2 roles, handling both Sv2
+message and Noise handshake frames.
+
+The Sv2 protocol is binary, with fixed message framing. Each message begins with the extension
+type, message type, and message length (six bytes in total), followed by a variable length
+message. The message framing is outlined below ([according to SV2 specs
+](https://stratumprotocol.org/specification/03-Protocol-Overview/#32-framing)):
+
```

Comment:
I would always use the github repo link to specs. In this case it would be: https://github.com/stratum-mining/sv2-spec/blob/main/03-Protocol-Overview.md#32-framing